transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java

```diff
@@ -45,10 +46,13 @@ private HyperwalletTransferMethodUi() {
     }
 
     /**
+     * @deprecated use {@link #getInstance(Context, HyperwalletAuthenticationTokenProvider)}
+     *
      * @param authenticationTokenProvider An implementation of the {@link HyperwalletAuthenticationTokenProvider}
      * @return Returns a newly created HyperwalletTransferMethodUi that can be used to get Intents to launch different
      * activities.
      */
+    @Deprecated
     public static synchronized HyperwalletTransferMethodUi getInstance(
```

Comment:
can we just delete it? If not, please create a ticket to remove the method and what are the pre conditions